app/services/post_out_of_sync_deploys_service.rb

```diff
@@ -28,6 +28,6 @@ def app_list(apps)
   end
 
   def app_info(app)
-    "- <https://github.com/#{app[:repo]}/actions/workflows/deploy.yml|#{app[:name]}> – #{app[:status].to_s.humanize}"
+    "- <#{Plek.find('release')}/applications/#{app[:shortname]}|#{app[:name]}> – #{app[:status].to_s.humanize} (<https://github.com/#{app[:repo]}/actions/workflows/deploy.yml|ship it!>)"
```

Comment:
Is "ship it!" the most appropriate text here? Probably better to change to "Repo" or "GitHub"?